### ISSUE TITLE
Missing link to JSON field on Fields Overview page

### ISSUE THREAD

**Issue by brachypelma** [payloadcms/payload#4988]
# Documentation Issue

The fields overview documentation page has [a list of field types](https://payloadcms.com/docs/fields/overview#field-types), but the [JSON field](https://payloadcms.com/docs/fields/json) is not listed.

Is there a way for me to edit the documentation? If not, could you please add a link?

**Comment by DanRibbens** [comment]
Yes, if you want to change the docs fork this repo and make changes to the mdx files in `/docs` and open a pull request.

**Comment by brachypelma** [comment]
Thank you, @DanRibbens. Pull request here: https://github.com/payloadcms/payload/pull/4989

**Comment by denolfe** [comment]
Thanks! Closed by #4989
